### PR TITLE
chore: release-it workflow

### PR DESCRIPTION
**Pull request by axi92**

### CODE REVIEW

File: .release-it.json [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
   },
   "github": {
-    "release": true
+    "release": true,
+    "assets": ["*.tgz"]
   },
   "plugins": {
     "@release-it/conventional-changelog": {
-      "preset": "angular"
+      "preset": "angular",
```
Comment (mhochsto) [inline .release-it.json:23]: Can we change this to react native ?

Comment (axi92) [reply .release-it.json:23]: I don't think that option exists https://github.com/release-it/conventional-changelog?tab=readme-ov-file#preset

Comment (axi92) [reply .release-it.json]: I redid the config, you can check it now

**Comment by axi92** [comment]
We can only use it when its on main. Can't test it on the branch unfortunately.

**Review by mhochsto — APPROVED** [review]

**Review by codepushr — APPROVED** [review]
Whatever 🍺

**Pull request merged**